derive/src/multihash.rs

```diff
@@ -253,11 +245,22 @@ pub fn multihash(s: Structure) -> TokenStream {
                 }
             }
 
-            fn digest_reader<R: #io_path::Read>(&self, reader: &mut R) -> #io_path::Result<Multihash> {
-                use #io_path;
+            #[cfg(feature = "std")]
```

Comment:
This gets expanded in the importing crate, not in the multihash-derive crate. That means if the importing crate doesn't _have_ an `std` feature, we'll use `core2`.

Reply:
I changed it so that it is expanded at the multihash-derive level. Would that work?